spec/Generics.tex

```diff
@@ -819,6 +803,7 @@ \subsection{User-Defined Constructors}
     this.p1 = p1;
     this.c1 = c1;
     this.v1 = v1;
+    // compiler inserts initialization for remaining fields
```

Comment:
I recognize you inherited this, but I think technically `c5` and `v5` are not generic.

I'm confused as to why they are suddenly named using 5, but I don't think that's important at all

Reply:
Going to leave this for another time, in the interest of the freeze deadline.